beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -38,6 +39,934 @@ const (
 	broadcastValidationConsensusAndEquivocation = "consensus_and_equivocation"
 )
 
+type handled bool
+
+// GetBlock retrieves block details for given block ID.
+//
+// DEPRECATED: please use GetBlockV2 instead
```

Comment:
now it's please use GetBlockV3

Reply:
there is no GetBlock V3 :smiley:

Reply:
it's called `ProduceBlockV3` now but it's the next iteration on top of getblock
https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Validator/produceBlockV3

Reply:
v2 is "deprecated" https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Validator/produceBlockV2 but we are still using it i believe

Reply:
I got confused with a different endpoint oops